test/e2e/storage/volume_provisioning.go

```diff
@@ -191,99 +130,6 @@ var _ = utils.SIGDescribe("Dynamic Provisioning", func() {
 						framework.ExpectNoError(err, "checkGCEPD pd-standard")
 					},
 				},
-				// AWS
-				{
-					Name:           "gp2 EBS on AWS",
-					CloudProviders: []string{"aws"},
-					Timeouts:       f.Timeouts,
-					Provisioner:    "kubernetes.io/aws-ebs",
-					Parameters: map[string]string{
-						"type": "gp2",
-						"zone": getRandomClusterZone(ctx, c),
-					},
-					ClaimSize:    "1.5Gi",
-					ExpectedSize: "2Gi",
-					PvCheck: func(ctx context.Context, claim *v1.PersistentVolumeClaim) {
-						volume := testsuites.PVWriteReadSingleNodeCheck(ctx, c, f.Timeouts, claim, e2epod.NodeSelection{})
-						gomega.Expect(volume).NotTo(gomega.BeNil(), "get bound PV")
-
-						err := checkAWSEBS(volume, "gp2", false)
-						framework.ExpectNoError(err, "checkAWSEBS gp2")
```

Comment:
I think only `checkAWSEBS()` function and these two lines should be removed - we still want to test that `type: gp2` in a StorageClass provisions a valid RWO volume. Without AWS SDK we can't check that it's really gp2, but IMO it's still better than nothing. The same applies to all AWS tests below.

Reply:
@jsafrane ack, will do this.